examples/tutorials/basic/your_first_hydra_app/3_using_config/default_resolver.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from omegaconf import DictConfig
+
+import hydra
+
+
+@hydra.main(config_name="resolver")
```

Comment:
this should not be in the tests for the tutorial (because it does not belong in the tutorial)